packages/spacecat-shared-data-access/src/service/import-url/accessPatterns.js

```diff
@@ -96,3 +96,23 @@ export const getImportUrlsByJobIdAndStatus = async (dynamoClient, config, log, j
   });
   return items.map((item) => ImportUrlDto.fromDynamoItem(item));
 };
+
+/**
+ * Get Import Urls by Job ID
+ * @param {DynamoClient} dynamoClient
+ * @param {Object} config
+ * @param {Logger} log
+ * @param {string} jobId
+ * @returns {Promise<ImportUrlDto[]>}
```

Comment:
Looks to me like `createImportUrl` returns a new ImportUrl, as opposed to the DTO.
```suggestion
 * @returns {Promise<ImportUrl[]>}
```

Reply:
JSDoc for `getImportUrlsByJobIdAndStatus` may need to be updated too.

Reply:
Done